actions/apps/explorer.sh

```diff
@@ -14,11 +14,27 @@ explorer_install ()
 
     info "Installing dependencies..."
     yarn install | tee -a "$commander_log"
-    success "Installed dependencies!"
+
+    check=${PIPESTATUS[0]}
```

Comment:
Should be local if it isn't re-used outside this function.

Reply:
Changed accordingly